market_nextdoor_api/serializers.py

```diff
@@ -36,4 +36,24 @@ class Meta:
     fields = ['id', 'customer', 'item', 'packed', 'fulfilled', 'ready', 'quantity_requested', 'vendor_id', 'date_created', 'updated_at', ]
 
   def get_vendor_id(self, obj):
-    return obj.item.vendor_id 
\ No newline at end of file
+    return obj.item.vendor_id 
+  
+
+# ManytoMany Test Serializer
+class Preorder_testItemSerializer(serializers.ModelSerializer):
```

Comment:
Is there a reason why we would want to serialize this? Are we ever going to return the preorder_item object to the Front end or will they just query it for a list of all the items on the preorder? I could definitely be overthinking this aspect.

Reply:
oh I think I can maybe answer my own question from looking at the below, serializing isn't just for the response, but for object creation in the db, correct?

Reply:
We aren't really creating in the serializer, but I built the Preorder_testItem serializer because the items are nested in the Preorder_test serializer. Preorder_testSerializer calls the item serializer and accesses Items within the fields.